apps/accounts/urls.py

```diff
@@ -5,4 +5,14 @@
 urlpatterns = [
     url(r"^user/disable$", views.disable_user, name="disable_user"),
     url(r"^user/get_auth_token$", views.get_auth_token, name="get_auth_token"),
+    url(
+        r"^challenge/invitation/(?P<challenge_pk>[0-9]+)/$",
```

Comment:
Please change the URL to `r"^(?P<challenge_pk>[0-9]+)/send-invitation/$",`

Can we also move this to challenges app ?